packages/transaction-controller/CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [36.2.0]
+
+### Changed
+
+- Bump `@metamask/controller-utils` from `^11.2.0` to `^11.3.0` ([#4688](https://github.com/MetaMask/core/pull/4688))
```

Comment:
Apologies for the hassle, but we'll need to add the change log entries for the startup fixes also, thanks!

https://github.com/MetaMask/core/pull/4658